cmd/rook/admission.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/rook/rook/cmd/rook/rook"
+	"github.com/rook/rook/pkg/operator/admission"
+	"github.com/rook/rook/pkg/util/flags"
+	"github.com/spf13/cobra"
+)
+
+var (
+	admissionCmd = &cobra.Command{
+		Use:   "admission-controller",
+		Short: "Starts admission controller",
+	}
+
+	storageSystem string
+)
+
+func init() {
+	admissionCmd.Flags().StringVar(&storageSystem, "storage", "ceph", "Starts the admission controller for corresponding storage")
+	flags.SetFlagsFromEnv(admissionCmd.Flags(), rook.RookEnvVarPrefix)
+	admissionCmd.RunE = startAdmissionController
+}
+
+func startAdmissionController(cmd *cobra.Command, args []string) error {
```

Comment:
The function signature returns an `error` but we never check for errors. Return nothing instead?